lib/generateSource.ts

```diff
@@ -15,16 +25,17 @@ export function generateSource(abi: Array<RawAbiDefinition>, context: IContext):
 
 // @todo better typings for web3
 function codeGenForContract(abi: Array<RawAbiDefinition>, input: Contract, context: IContext) {
-  const typeName = `${context.fileName}`;
-  return `/* GENERATED BY TYPECHAIN VER. ${getVersion()} */
+  const runtimeNamespace = "TC";
+  const typeName = context.fileName;
+
+  return `
+/* GENERATED BY TYPECHAIN VER. ${getVersion()} */
 /* tslint:disable */
   
 import { BigNumber } from "bignumber.js";
-import { TypeChainContract, promisify, ITxParams, IPayableTxParams, DeferredTransactionWrapper, DeferredEventWrapper } from '${
-    context.relativeRuntimePath
-  }'
+import * as ${runtimeNamespace} from "${context.relativeRuntimePath}"
```

Comment:
@krzkaczor I'm not a fan of importing entire module thought `* as ...` but I found it the easiest approach to fix a bug.

Reply:
Yeah, I like this solution 👍